### PR TITLE
Server-priority label

### PR DESCRIPTION
**Pull request by Safihre**
I made the label, @shypike will you actually sort them on server-priority in config-view?

### CODE REVIEW

File: sabnzbd/interface.py [hunk 1]
```diff
@@ -1501,13 +1501,6 @@ def add_color(dir, color):
                 else:
                     wlist.append(rweb)
         conf['web_list'] = wlist
-
-        # Obsolete template variables, must exist and have a value
-        conf['web_colors'] = ['None']
-        conf['web_color'] = 'None'
-        conf['web_colors2'] = ['None']
-        conf['web_color2'] = 'None'
-
```
Comment (shypike) [inline sabnzbd/interface.py]: Are you sure these can go? No templates use them?

Comment (Safihre) [inline sabnzbd/interface.py]: Since we use the uniconfig now for all skins, this is not used anymore.

**Comment by shypike** [comment]
I cannot sort in interface.py, because it passes a dictionary of servers, which is inherently unsorted.
So either I start sending a list, otherwise you need to sort the dictionary in the skin.

**Comment by thezoggy** [comment]
just use ordered dict?

**Comment by shypike** [comment]
No, I'll send the server as a list so that the Python code determines the order. Changing the skin code is fairly easy to do, but it needs to be done in sync.

**Comment by Safihre** [comment]
Adding search functionality to the queue.

**Comment by shypike** [comment]
I also added sorting the server on priority and name (in that order).

**Comment by thezoggy** [comment]
@shypike @Safihre 

noticed that glitter status and config servers have different order:

config > servers:
sab3

glitter > status:
sab4

**Pull request merged**